release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -494,6 +514,20 @@ With this release, the Performance Addon Operator (PAO) must-gather image is no
 
 For further information about gathering debugging information relating to low-latency tuning, see xref:../scalability_and_performance/cnf-low-latency-tuning.adoc#cnf-collecting-low-latency-tuning-debugging-data-for-red-hat-support_cnf-master[Collecting low latency tuning debugging data for Red Hat Support].
 
+[id="ocp-4-14-NRO-image-must-gather"]
+==== Collecting data for the NUMA Resources Operator with the `must-gather` image of the operator
+
+In this release, the `must gather` tool is updated to collect the data of the NUMA Resources Operator with the `must-gather` image of the operator.
+
+For further information about gathering debugging information relating to the NUMA Resources Operator, see xref:../scalability_and_performance/cnf-numa-aware-scheduling.html#cnf-about-collecting-nro-data_numa-aware[Collecting NUMA Resources Operator data].
+
+[id="ocp-4-14-additional-power-savings-control"]
+==== Enabling more control over the c-states for each pod
+
+With this release, users are given more control over the c-states for their pods. Now instead of just disabling c-states completely users are able to specify a maximum latency in microseconds for the C-states. The additional configuration is available in the `cpu-c-states.crio.io` annotation and facilitates power savings in the high-priority applications by enabling some of the shallower c-states, instead of disabling them completely.
```

Comment:
Watch comma use.

```suggestion
With this release, users are given more control over the c-states for their pods. Now, instead of disabling c-states completely, users are able to specify a maximum latency in microseconds for the C-states. The additional configuration is available in the `cpu-c-states.crio.io` annotation and facilitates power savings in the high-priority applications by enabling some of the shallower c-states, instead of disabling them completely.
```

Inconsistent capitalization of "c-states"/"C-states".